samples/unity-of-bugs/Assets/Scripts/DebounceTester.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DebounceTester : MonoBehaviour

```

Comment:
If this is just for testing (not for demo purposes), can this live in the Tests project instead?

Reply:
It was needed for my demos. I'll remove it, not needed.